decoder/nginx.go

```diff
@@ -0,0 +1,39 @@
+package decoder
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"time"
+
+	insaneJSON "github.com/vitkovskii/insane-json"
+)
+
+const (
+	nginxDateFmt = "2006/01/02 15:04:05"
+)
+
+// DecodeNginxError decodes nginx error log format. Like:
+// 2022/08/17 10:49:27 [error] 2725122#2725122: *792412315 lua udp socket read timed out, context: ngx.timer
+func DecodeNginxError(event *insaneJSON.Root, data []byte) error {
```

Comment:
Can we parse other fields as well??
`YYYY/MM/DD HH:MM:SS [LEVEL] PID#TID: *CID MESSAGE`

Reply:
> and logs are always ERROR

https://nginx.org/en/docs/ngx_core_module.html#error_log

> the level of logging, and can be one of the following: debug, info, notice, warn, error, crit, alert, or emerg